themes/label-5.yaml

```diff
@@ -26,11 +26,11 @@ global:
     text_visible_trunk_primary_route:   true        # true by default
     text_visible_trunk_primary_e:       false     # false for default, false for more
     text_visible_secondary:             true
-    text_visible_secondary_e:           false     # false for default, false for more
+    text_visible_secondary_e:           true     # false for default, false for more
     text_visible_tertiary:              true
     text_visible_tertiary_e:            false     # false for default, false for more
     text_visible_minor_road:            true
-    text_visible_minor_road_e:          true
+    text_visible_minor_road_e:          false
```

Comment:
If these are no longer referenced (other code changes removed them, but require further discussion) consider removing them here from list of globals.